File: 1015.py
#1015 DISTÂNCIA ENTRE DOIS PONTOS
import math

ValoresP1 = input().split(" ")
ValoresP2 = input().split(" ")

p1 = [float(valor) for valor in ValoresP1]
p2 = [float(valor) for valor in ValoresP2]

distancia = math.sqrt((p2[0] - p1[0])**2 + (p2[1] - p1[1])**2)
print("{:.4f}" .format(distancia))

File: javascript/1015.js
/*Distância Entre Dois Pontos*/
const input = require("fs").readFileSync("/dev/stdin", "utf8");

const lines = input.split("\n");

const [x1, y1] = lines[0].split(" ");
const [x2, y2] = lines[1].split(" ");

const calculo = (((x2 - x1)**2) + ((y2 - y1)**2))
const Distancia = Math.sqrt(calculo, 2);

console.log(Distancia.toFixed(4));